python/bindings.cpp

```diff
@@ -543,13 +543,28 @@ PYBIND11_MODULE(_libsonata, m) {
         m, "NodePopulationProperties", "Stores node population-specific network information")
         .def_readonly("spatial_segment_index_dir",
                       &NodePopulationProperties::spatialSegmentIndexDir,
-                      DOC_NODE_POPULATION_PROPERTIES(spatialSegmentIndexDir));
+                      DOC_NODE_POPULATION_PROPERTIES(spatialSegmentIndexDir))
+        .def_readonly("vasculature_file",
+                      &NodePopulationProperties::vasculatureFile,
+                      DOC_NODE_POPULATION_PROPERTIES(vasculatureFile))
+        .def_readonly("vasculature_mesh",
```

Comment:
This should be "vasculature_mesh_file"

Reply:
yup, I'll change that; can you just use this name for the test?

Reply:
My bad. The test config is old. vasculature_mesh is the correct term, being also in the spec.